e2e/specs/lab-orders.spec.ts

```diff
@@ -1,93 +1,51 @@
 import { expect } from '@playwright/test';
 import { type Visit } from '@openmrs/esm-framework';
-import { generateRandomPatient, type Patient, startVisit, endVisit, deletePatient } from '../commands';
+import { type Order } from '@openmrs/esm-patient-common-lib';
+import { type Encounter, type Provider } from '../commands/types';
+import {
+  generateRandomPatient,
+  type Patient,
+  generateRandomTestOrder,
+  deleteTestOrder,
+  createEncounter,
+  deleteEncounter,
+  getProvider,
+  startVisit,
+  endVisit,
+  deletePatient,
+} from '../commands';
 import { test } from '../core';
 import { OrdersPage } from '../pages';
 
 let patient: Patient;
 let visit: Visit;
+const url = process.env.E2E_BASE_URL;
+let fullName: String;
+let drugOrder: Order;
+let encounter: Encounter;
+let orderer: Provider;
 
 test.beforeAll(async ({ api }) => {
   patient = await generateRandomPatient(api);
```

Comment:
We should probably have a patient-per-test rather than one patient across all tests.